base/complex.jl

```diff
@@ -275,6 +275,7 @@ muladd(z::Complex, w::Complex, x::Complex) =
 # handle Bool and Complex{Bool}
 # avoid type signature ambiguity warnings
 +(x::Bool, z::Complex{Bool}) = Complex(x + real(z), imag(z))
++(z::Complex{Bool}) = Complex(+z.re , +z.im)
```

Comment:
This is kind of a weird place for this. It's also weird that it's specifically for `Complex{Bool}`—this definition should apply to any kind of `Complex`. I would not put it here and I would broaden it.
```suggestion
```
Instead, I would put it on line 265 like this:
```jl
+(z::Complex) = Complex(+real(z), +imag(z))
-(z::Complex) = Complex(-real(z), -imag(z))
```

Reply:
Cool